CHANGELOG.rst

```diff
@@ -19,6 +21,9 @@ Releases
   ``FunctionSpec``.
 - ``ServiceSpec`` now provides a ``lookup`` method to look up ``FunctionSpecs``
   by name.
+- Removed the ``force`` option on ``Loader.load``.
+- In generated modules, renamed the ``types``, ``constants`` and ``services``
```

Comment:
This is potentially breaking, but I buy the argument that these were intended to be internal types.